### PR TITLE
Add sentry-ruby 4.5.0's new config options

### PR DESCRIPTION
**Pull request by st0012**

### CODE REVIEW

File: src/platforms/ruby/common/configuration/options.mdx [hunk 1]
```diff
@@ -152,6 +152,14 @@ Sentry respects logger levels.
 config.max_breadcrumbs = 30
 ```
 
+`propagate_traces`
+
+: By default, Sentry injects `sentry-trace` header to outgoing requests made with `Net::HTTP` to connect traces between services. You can disable this behavior with
```
Comment (imatwawana) [inline src/platforms/ruby/common/configuration/options.mdx:157]:
```suggestion
: By default, Sentry injects the `sentry-trace` header in outgoing requests made with `Net::HTTP` to connect traces between services. You can disable this behavior with
```

**Review by imatwawana — APPROVED** [review]
Minor wording edits

**Review by HazAT — APPROVED** [review]

**Comment by HazAT** [comment]
Damn sorry @imatwawana I just clicked merge at the same time 😅

**Comment by imatwawana** [comment]
No worries @HazAT! They weren't earth shattering changes. ;-)

**Comment by st0012** [comment]
@HazAT @imatwawana thanks to both of you 😂 should I open another PR for the wording?

**Comment by imatwawana** [comment]
Naw, don't worry about it! 😁

**Pull request merged**